File: OddNumbers.py
# define a class
class OddNumerals:

    ###
     # Our constructor.
     # @param alpha - for the start value
     # @param omega - for the end value
     ##
    def __init__(self, alpha, omega):
        self.start = alpha
        self.stop = omega
        self.result = [] # list to hold our answers


    # Returns an list of the desired set of odd numbers
    def prepResult(self):
        # Loop from start to stop and rip out odd numbers;
        self.i = 0
        while self.start <= self.stop:
            if self.start % 2 != 0:
                self.result.append(self.start)
                self.i += 1
                
            self.start = self.start + 1 # increase start by 1
            
        return self.result

File: OddNumbers.cs
using System.Collections.Generic;

namespace Arithmetic_CS
{
    class OddNumbers
    {
        private int start; // Our starting point
        private int stop; // where we will stop
        private List<int> list_of_primes; // We will house our gathered prime numbers here.

        // Our constructor
        public OddNumbers(int first, int last)
        {
            start = first;
            stop = last;
            list_of_primes = new List<int>();
        }

        public List<int> prepResult()
        {
            /*
            * Loop from start to stop and rip out even numbers;
             */
            while (start <= stop)
            {
                if (start % 2 != 0)
                {
                    list_of_primes.Add(start);
                }
                start++; // increase 'start' by 1
            }
            return list_of_primes;
        }
    }
}
